config/doctrine-encrypt-module.global.php.dist

```diff
@@ -8,31 +8,27 @@
  * You may want to define a different file for development / production
  */
 return array(
-    'doctrine' => array (
-        'encryption' => array (
-            'orm_default' => array (
+    'doctrine' => array(
+        'encryption' => array(
+            'orm_default' => array(
 
                 //The adapter can be a reference to another service locator key, a Fully qualified class name
                 //or an anonymous function. However, it must implement \Zend\Crypt\BlockCipher or
                 //DoctrineEncrypt\Encryptors\EncryptorInterface
-                'adapter' => function() {
-                    $cipher = \Zend\Crypt\BlockCipher::factory('mcrypt');
-                    $cipher->setKey('Test Key');
-                    return $cipher;
-                },
+                # 'adapter' => \DoctrineEncryptModule\Factory\McryptAdapter::class,
 
                 //The reader will read encryption configuration information off of the entities
                 //You probably don't have to change this
-                'reader' => 'Doctrine\Common\Annotations\AnnotationReader',
-            )
+                # 'reader' => 'Doctrine\Common\Annotations\AnnotationReader',
+            ),
         ),
 
         'eventmanager' => array(
```

Comment:
If we move the event manager definition to the module.config.php is there any point in even keeping this file?